clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/TransferArraysV1Key.scala

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.clio.transfer.model.arrays
+
+import org.broadinstitute.clio.transfer.model.TransferKey
+import org.broadinstitute.clio.util.model.Location
+
+case class TransferArraysV1Key(
+  location: Location,
+  chipwellBarcode: String,
+  analysisVersionNumber: Int,
+  version: Int
+) extends TransferKey {
+
+  override def getUrlSegments: Seq[String] =
+    Seq(
+      location.entryName,
+      chipwellBarcode,
```

Comment:
We may want to separate chip barcode from chip position. It could help with chip based queries. Thoughts?

Reply:
Please explain more, Jay.

Reply:
chip well barcode is actually and combination of the chip barcode and the well position.  so 123456789_R01C02 is chip barcode 123456789 row 01 column 02

If someone noticed something was wrong with a sample they might want to go back and look at other samples on that same chip. So we could either keep chipwell barcode as is and look for all keys that are like 123456789_* or we could separate the chip barcode from the well position and just search for all samples with a chip barcode of 123456789.